package.json

```
@@ -24,9 +24,9 @@
     "semantic-release": "semantic-release"
   },
   "dependencies": {
-    "@typescript-eslint/utils": "^6.0.0",
+    "@typescript-eslint/utils": "^6.0.0 || ^7.0.0",
```

Since this is a direct dependency it's better to stick with one version, let's just upgrade it to v7 and drop v6:
```suggestion
    "@typescript-eslint/utils": "^7.0.0",
```